userspace/falco/lua/output.lua

```diff
@@ -152,70 +169,86 @@ end
 function mod.http_reopen()
 end
 
+function mod.grpc(event, rule, source, priority, priority_num, msg, format, options)
```

Comment:
I think it's fine to do it this way, but I always thought routing events through lua just so they could be written to a log file/stdout/back to c for http output was a bit awkward. Did you consider having grpc outputs entirely separate from lua and using the json_event_formatter directly?

Reply:
I know, the purpose of this was to keep things as simple as possible as first implementation. We'd like to do that as a refactoring.